mqtt-broker/mqtt-broker/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "mqtt-broker"
+version = "0.1.0"
+authors = ["Azure IoT Edge Devs"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+async-trait = "0.1"
+failure = "0.1"
+futures-util = { version = "0.3", features = ["sink"] }
+lazy_static = "1"
+tokio = { version = "0.2", features = ["macros", "rt-threaded", "signal", "stream", "sync", "tcp"] }
+tokio-io-timeout = "0.4"
+tokio-util = { version = "0.2", features = ["codec"] }
+tracing = "0.1"
+tracing-futures = "0.2"
+uuid = { version = "0.8", features = ["v4"] }
+
+mqtt = { git = "https://github.com/myagley/mqtt", branch = "v0.2.x" }
```

Comment:
You should import this too.

Reply:
Like, put it next to the broker code locally?

Reply:
done